build_targets

```diff
@@ -6,6 +6,7 @@ bladerunner userdebug yes no
 courbet user yes yes
 ginkgo userdebug yes yes
 guacamole userdebug yes yes
+guacamoleb userdebug yes yes
```

Comment:
dt and kernel should not be tracked from LOS

Reply:
they were being tracked from LOS since they didn't have any modifications, i'll upload and switch to crdroidandroid github whenever i have free time